src/haddock/modules/topology/topoaa/__init__.py

```diff
@@ -202,7 +202,7 @@ def _run(self):
                 try:
                     model_id = int(model.stem.split('_')[-1])
                     md5_hash = md5_dic[model_id]
-                except ValueError:
+                except Exception:
```

Comment:
I suggest specifying here the exact errors that should result in `md5_hash = None`. Also, I suggest having a single statement inside the `try` block.